### DISCUSSION TITLE
Suggest relevant discussions and answers

### DISCUSSION THREAD

**Discussion by vanbroup** [community/community#3291] [Discussions]
Automatically showing relevant discussions and answers when creating a new discussion could help against the creation of duplicate discussions and reduce the load on project maintainers.

I imagine that this could be shown on the right under the `Helpful resources` section.

**Comment by PranamBhat** [comment]
Hello @vanbroup 

**GitHub** Discussion page already has enough filters at the top page.

If GitHub adds another section "**Helpful resources**", it would be more into mess rather than giving required data!
